modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/impl/RaftGroupServiceImpl.java

```diff
@@ -214,88 +221,134 @@ private RaftGroupServiceImpl(
         sendWithRetry(leader, req, currentTimeMillis() + timeout, fut);
 
         return fut.thenApply(resp -> {
-            peers = resp.peers();
-            learners = resp.learners();
+            peers = parsePeerList(resp.peersList());
+            learners = parsePeerList(resp.learnersList());
```

Comment:
If there are no learners are we going to return empty list or null from resp.learnersList()?

Reply:
Empty list will be returned according to GetPeersRequestProcessor